packages/com.rawrlab.pmouya.yml

```diff
@@ -0,0 +1,8 @@
+title: eXplosive Dinosaurs (Demo)
+iconUri: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/icon160.png
+manifestUrl: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/com.rawrlab.pmouya.manifest.json
+category: games
+description: |

```

Comment:
This is a very nice game, please consider to add more screenshots, and information in the description field. [Example](https://repo.webosbrew.org/apps/com.limelight.webos)